SPEC.md

```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Abbreviated reference as `acName`.
```

Comment:
sorry, this doesn't make sense; `acName` isn't mentioned anywhere else in the spec, and the other kinds are not references but field names. IMHO just remove this and 565 / 578